compiler/sem/semstmts.nim

```diff
@@ -2138,14 +2128,13 @@ proc semAllTypeSections(c: PContext; n: PNode): PNode =
   proc gatherStmts(c: PContext; n: PNode; result: PNode) {.nimcall.} =
     case n.kind
     of nkIncludeStmt:
-      for i in 0 ..< n.len:
-        var f = checkModuleName(c.config, n[i])
+      for p in n.items:
```

Comment:
If you're modifying `semAllTypeSections`, it might make sense to instead remove it, as it's currently unused.

Reply:
The haphazard formatting was really bothering me as I was scrolling past it. But I totally missed the fact that it wasn't used, wow.

Reply:
No worries, I just thought it's worth mentioning.

Reply:
Removed it and the associated legacy report, thanks.

(would have done it yesterday but had a headache 😞)